sdformat_mjcf/sdformat_mjcf/converters/geometry.py

```diff
@@ -142,6 +147,6 @@ def add_visual(body, vis, pose_resolver=su.pose_resolver):
     """
     sem_pose = vis.semantic_pose()
     pose = pose_resolver(sem_pose)
-    geom = add_geometry(body, vis.name(), pose, vis.geometry())
+    geom = add_geometry(body, vis.name(), pose, vis.geometry(), material)
```

Comment:
I'm not sure if this function should take a `material` argument. Shouldn't it
convert the `<material>` inside the SDFormat `<visual>`?